pkg/secrets/secrets.go

```diff
@@ -187,9 +183,63 @@ func SecretMounts(filePath, mountLabel, containerWorkingDir string) ([]rspec.Mou
 
 		mounts = append(mounts, m)
 	}
+
+	// Add FIPS mode secret
+	// The filePath check is to ensure that this secret is created only once
+	// as SecretMounts is called twice for the two mounts.conf files ("OverrideMountsFile" and "DefaultMountsFile") in the respective order
+	// therefore we want to add the FIPS mode secret, if it exists, after the secrets from the DefaultMountsFile are mounted
+	_, err := os.Stat("/etc/system-fips")
```

Comment:
Should check if err != NotExists and return an error.

Reply:
Why return an error? We are just checking if the host has the "system-fips" file, if it doesn't we move on. I don't see why we should fail.